nullaway/src/test/java/com/uber/nullaway/NullAwayJSpecifyArrayTests.java

```diff
@@ -0,0 +1,88 @@
+package com.uber.nullaway;
+
+import com.google.errorprone.CompilationTestHelper;
+import java.util.Arrays;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class NullAwayJSpecifyArrayTests extends NullAwayTestsBase {
+
+  @Test
+  public void arrayDimensionAnnotationDereference() {
+    makeHelper()
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import org.jspecify.annotations.Nullable;",
+            "class Test {",
+            "  static Integer @Nullable [] fizz = {1};",
+            "  static void foo() {",
+            "     // BUG: Diagnostic contains: dereferenced expression fizz is @Nullable",
+            "     int bar = fizz.length;",
+            "  }",
+            "}")
+        .doTest();
+  }
+
+  @Test
+  public void arrayDimensionAnnotationAssignment() {
+    makeHelper()
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import org.jspecify.annotations.Nullable;",
+            "class Test {",
+            "  Object foo = new Object();",
+            "  void m( Integer @Nullable [] bar) {",
+            "      // BUG: Diagnostic contains: assigning @Nullable expression to @NonNull field",
+            "      foo = bar;",
+            "  }",
+            "}")
+        .doTest();
+  }
+
+  @Ignore("Array type annotations aren't supported currently.")
+  @Test
+  public void arrayTypeAnnotationDereference() {
+    makeHelper()
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import org.jspecify.annotations.Nullable;",
+            "class Test {",
+            "  static @Nullable String [] fizz = {\"1\"};",
+            "  static void foo() {",
+            "      // BUG: Diagnostic contains: dereferenced expression fizz[0] is @Nullable.",
+            "      int bar = fizz[0].length();",
+            "  }",
+            "}")
+        .doTest();
+  }
+
+  @Ignore("Array type annotations aren't supported currently.")
```

Comment:
Same comment as above about not ignoring

Reply:
Fixed